extensions/lifecycle/tasks/LifecycleTask.js

```diff
@@ -828,7 +829,9 @@ class LifecycleTask extends BackbeatTask {
               .setAttribute('target.key', params.objectKey)
               .setAttribute('target.version', params.encodedVersionId)
               .setAttribute('target.eTag', params.eTag)
-              .setAttribute('toLocation', params.site);
+              .setAttribute('toLocation', params.site)
+              .setAttribute('details.lastModified', params.lastModified);
```

Comment:
minor point: I know this have been derived from object processor code, but I think it's more logical to put this into `target` as it's directly related to defining target properties. It could make the message organization slightly clearer in case we have to add more stuff in the future.